doc/appendix-de-kassensichv/examples/pos-receipt.md

```diff
@@ -0,0 +1,468 @@
+      "Moment": "2019-10-25T13:32:45.133Z"
+    },
+    {
+      "Position": 2,
+      "Quantity": 1.0,
+      "Description": "Schnitzel",
+      "Amount": 9.20,
+      "VATRate": 7.00,
+      // 0x4445000000000012 (delivery reduced-1)
+      "ftChargeItemCase": 4919338167972134930,
+      "ftChargeItemCaseData": "",
+      "VATAmount": 0.60,
+      "CostCenter": "1",
+      "AccountNumber": "Accountnumber",
+      "ProductGroup": "Speisen",
+      "ProductNumber": "102",
+      "ProductBarcode": "",
+      "Unit": "Stk",
+      "UnitQuantity": 1,
+      "UnitPrice": 9.20,
+      "Moment": "2019-10-25T13:32:45.133Z"
+    }
+  ],
+  "cbPayItems": [
+    {
+      "Position": 1,
+      "Quantity": 1,
+      "Description": "Bar",
+      "Amount": 13.0,
+      // 0x4445000000000001 (cash payment)
+      "ftPayItemCase": 4919338167972134913,
+      "ftPayItemCaseData": "",
+      "AccountNumber": "Accountnumber",
+      "CostCenter": "Costcenter123",
+      "MoneyGroup": "BAR",
+      "MoneyNumber": "1234",
+      "Moment": "2019-10-25T13:32:45.133Z"
+    }
+  ],
+  // 0x4445 0000 0000 0001 (pos-receipt) 
+  "ftReceiptCase": 4919338167972134913,
+  "ftReceiptCaseData": "",
+  "cbReceiptAmount": 13.00,
+  "cbUser": "Chef",
+  "cbArea": "Tisch 01",
+  "cbSettlement": "Test1"
+}
+`
+
+
+ReceiptResponse
+
+`
+{
+    "ftCashBoxID": "cashboxid-guid",
+    "ftQueueID": "queueid-guid",
+    "ftQueueItemID": "queueitemid-guid",
+    "ftQueueRow": 96,
+    "cbTerminalID": "1",
+    "cbReceiptReference": "1",
+    "ftCashBoxIdentification": "[ftCashBoxIdentification]",
+    "ftReceiptIdentification": "ft[ftQueueRow]#T[finishTransactionNr]",
+    "ftReceiptMoment": "2019-10-25T13:32:46.133Z",
+    "ftSignatures": [
+        {
+            "ftSignatureFormat": 3,
+            //0x4445000000000001 (qr-code according kasssichv)
+            "ftSignatureType": 4919338167972134913,
+            "caption": "www.fiskaltrust.de",
+            "data": "V0;[ftCashBoxIdentification];Kassenbeleg-V1;Beleg^3.80_9.20_0.00_0.00_0.00^13.00:Bar;[finishTransactionNr];[signatureCounter];[startTransactionMoment];[finishTransactionMoment];[signatureAlgorithm];[logTimeFormat e.g. unixtime];[signature];[publickey]"
+        },
+        {
+            "ftSignatureFormat":13,
+            //0x4445000000000010 (start-transaction-result)
+            "ftSignatureType": 4919338167972134928,
+            "caption": "start-transaction-signature",
+            "data": "[startTransactionResult]"
+        },
+        {
+            "ftSignatureFormat":13,
+            //0x4445000000000011 (finish-transaction-payload)
+            "ftSignatureType": 4919338167972134929,
+            "caption": "finish-transaction-payload",
+            "data": "[finishTransactionPayload]"
+        },
+        {
+            "ftSignatureFormat": 13,
+            //0x4445000000000012 (finish-transaction-result)
+            "ftSignatureType": 4919338167972134930,
+            "caption": "finish-transaction-payload",
```

Comment:
I assume this will be called `finish-transaction-signature`

Reply:
`signature` is not enough in my point of view. no glue whats the format of the content here, because it needs to unify results of different kinds and brands of TSE.
minimum content in my point of view is
- moment of the signature
- signature-counter
- signature-value
- hash-value used to create signature (optional)